### ISSUE TITLE
Replace information modals with non-blocking solution

### ISSUE THREAD

**Issue by joshas** [fman-community/fman#183]
Currently tutorial and statistics collection popups are blocking fman UI. Would be nice if those messages would be displayed in non blocking way, e.g. as Firefox does displaying information on bottom of screen:
![refresh-notification-after-firefox-nigthly-reinstall](https://user-images.githubusercontent.com/79141/27514725-c780bbb4-599b-11e7-9f42-358d33b033b0.png)

**Comment by mherrmann** [comment]
I can see that it's a little annoying. The tutorial is currently not modal, in the technical sense, in that you can perform all operations in fman as usual. It's just that the tutorial is overlain over the GUI. I believe what you are saying is that you don't want the tutorial to hide the rest of the GUI (?). If yes, I'd be afraid that many first-time users will then miss the tutorial. I wouldn't want that to happen.

**Comment by joshas** [comment]
@mherrmann Point about tutorial sounds reasonable. I just wanted to point out possible alternative for modals, as they usually are quite annoying. E.g. new version notification works quite well, as it is not obtrusive.

**Comment by mherrmann** [comment]
@joshas yes, I see your point. I also really like the unobtrusiveness of the update notification. I'd like to keep the tutorial as it is for the discussed reason but maybe something can be done about the statistics popup.
